tidy3d/components/data/unstructured/base.py

```diff
@@ -255,6 +255,8 @@ def _values_coords_dict(self):
         """Non-spatial dimensions are corresponding coordinate values of stored data."""
         coord_dict = {dim: self.values.coords[dim].data for dim in self.values.dims}
         _ = coord_dict.pop("index")
+        if "voltage" in coord_dict.keys():
+            _ = coord_dict.pop("voltage")
```

Comment:
the purpose of `_values_coords_dict` is to return all dimensions other than `index`. Why did you need to remove `voltage` too?

Reply:
Yeah, I wasn't sure about this. I'll remove that